src/js/components/Select/stories/LazyLoading.js

```diff
@@ -55,39 +51,32 @@ class LazyLoading extends Component {
           sensitivity: 'base',
         });
       }),
-    });
+    );
   };
 
-  onChange = ({ selected: nextSelected }) => {
-    this.setState({ selected: nextSelected });
-  };
+  const onChange = ({ selected: nextSelected }) => setSelected(nextSelected);
+
+  return (
+    <Grommet theme={grommet}>
+      <Box fill align="center" justify="start" pad="large">
+        <Select
+          multiple
+          closeOnChange={false}
+          placeholder="select an option..."
+          selected={selected}
+          options={options}
+          dropHeight="medium"
+          onMore={onMore}
```

Comment:
I'm seeing this console error for the `onMore`
```
react-dom.development.js:545 Warning: Unknown event handler property `onMore`. It will be ignored.
    in input (created by Context.Consumer)
    in StyledComponent (created by StyledTextInput)
    in StyledTextInput (created by TextInput)
    in Keyboard (created by TextInput)
    in div (created by Context.Consumer)
    in StyledComponent (created by StyledTextInput__StyledTextInputContainer)
    in StyledTextInput__StyledTextInputContainer (created by TextInput)
    in TextInput (created by TextInput)
    in TextInput (created by Context.Consumer)
    in TextInput (created by Context.Consumer)
    in WithTheme(TextInput) (created by FocusableComponent)
    in FocusableComponent (created by WithTheme(TextInput))
    in WithTheme(TextInput) (created by Context.Consumer)
    in StyledComponent (created by Select__SelectTextInput)
    in Select__SelectTextInput (created by Select)
    in div (created by Context.Consumer)
    in StyledComponent (created by StyledBox)
    in StyledBox (created by Box)
    in Box (created by Box)
    in Box (created by Context.Consumer)
    in WithTheme(Box) (created by Select)
    in div (created by Context.Consumer)
    in StyledComponent (created by StyledBox)
    in StyledBox (created by Box)
    in Box (created by Box)
    in Box (created by Context.Consumer)
    in WithTheme(Box) (created by Select)
    in button (created by Context.Consumer)
    in StyledComponent (created by StyledButton)
    in StyledButton (created by Button)
    in Button (created by Button)
    in Button (created by Context.Consumer)
    in WithTheme(Button) (created by FocusableComponent)
    in FocusableComponent (created by WithTheme(Button))
    in WithTheme(Button) (created by DropButton)
    in DropButton (created by DropButton)
    in DropButton (created by Context.Consumer)
    in StyledComponent (created by Select__StyledSelectDropButton)
    in Select__StyledSelectDropButton (created by Select)
    in Keyboard (created by Select)
    in Select (created by Select)
    in Select (created by Context.Consumer)
    in WithTheme(Select) (created by LazyLoading)
    in div (created by Context.Consumer)
    in StyledComponent (created by StyledBox)
    in StyledBox (created by Box)
    in Box (created by Box)
    in Box (created by Context.Consumer)
    in WithTheme(Box) (created by LazyLoading)
    in div (created by Context.Consumer)
    in StyledComponent (created by StyledGrommet)
    in StyledGrommet (created by Grommet)
    in Grommet (created by LazyLoading)
    in LazyLoading
```